lib/phoenix_live_view/helpers.ex

```diff
@@ -671,6 +671,63 @@ defmodule Phoenix.LiveView.Helpers do
     end
   end
 
+  @doc """
+  Renders a slot entry with the given optional `argument`.
+
+      <%= render_slot(@footer, item: @item)
+
+  If multiple slot entries are defined for the same slot, `render_slot/2` will automatically
+  render all entries, merging their contents. In case you want to use the entries' attributes,
+  you need to iterate over the list to access each one of them individually.
+
+      <%= for col <- @cols do %>
+      <%= col.label %>: <%= render_slot(col) %>
+      <% end %>
+
+  """
+  defmacro render_slot(slot, argument \\ nil) do
+    quote do
+      unquote(__MODULE__).__render_slot__(
+        var!(changed, Phoenix.LiveView.Engine),
+        unquote(slot),
+        unquote(argument)
+      )
+    end
+  end
+
+  @doc false
+  defmacro slot(name, attrs, do: do_block) do
```

Comment:
Originally, the `name` here was meant to be the name of the slot. But we agreed that the name is not necessary, so this should be:

```suggestion
  defmacro slot(attrs, do: do_block) do
```

The block will always be `:inner_block`. Remember we can also have slots with no `inner_block` whatsoever, right? Do we keep it nil? If so, do we raise a good error message inside `render_slot`? Or does it always render an empty string?